docs/Account/methods.md

```diff
@@ -257,7 +257,7 @@ type SupportedToken = {
 };
 ```
 
-### [getGasEstimates()](https://bcnmy.github.io/biconomy-client-sdk/classes/BiconomySmartAccountV2.html#getGasEstimates)
+### [getGasEstimate()](https://bcnmy.github.io/biconomy-client-sdk/classes/BiconomySmartAccountV2.html#getGasEstimates)
```

Comment:
this should also be https://bcnmy.github.io/biconomy-client-sdk/classes/BiconomySmartAccountV2.html#getGasEstimate

Reply:
Updated the link but I'm not able to open it.